chcemvediet/settings/server_local.py

```diff
@@ -66,6 +66,10 @@
             u'handlers': [u'console'],
             u'level': u'INFO',
             },
+        u'chcemvediet': {
```

Comment:
@martinmacko47 Doteraz sa chyby z chcemvediet logovali iba v ramci `root` loggera, teraz sa bude rovnako ako pri `poleno` logger zapisovat na 2-krat. Ak chceme zamedzit tomuto spravaniu mozeme pouzit attribut `propagate: False`

Reply:
Co si pamatam, tak to propagate bolo nejak rozbite a nepropagovalo sa to do `root` loggera. Preto nelogujeme iba `root` ale explicitne vsetko. Aspon teda nevsimol som si, ze by bez `propagate: False` chodili duplicitne errory. Ak zacnu, poriesime potom.